### ISSUE TITLE
Core logic

### ISSUE THREAD

**Issue by arashmh** [cool-RR/grid_royale#5]
Hello there

Im very intrigued by the idea of this project. I was thinking that maybe the core logic of it is a bit too "biological" for being basis of "social" dynamics. 

Don't get me wrong in being sentimental, let me explain what i mean. Of course the struggle to avoid "death" is encoded in the nature of any biological and living thing. And many social behaviors have emerged solely out if the struggle to survive or effectiveness of living in social groups. 

But, joining a gang is a fatal decision that has been made by millions throughout history of mankind. The average age of death of gang members is 23 or something like that. The point is that decisions that individuals make might be recurring among individuals even if those decisions are guaranteed to kill you, one way or the other.

**Comment by cool-RR** [comment]
I'm happy you're intrigued by the project. I'm not sure what to do with your line of thinking though.

**Comment by arashmh** [comment]
Yeah I just wanted to discuss the idea and maybe even more similar thoughts to consolidate a body of perspectives and then translate those to the algorithmic tweaks needed.

**Comment by cool-RR** [comment]
Okay, I'm still in the opinion that all social behavior should be based on survival behaviors like seeking for foods. So if you want to push your idea, you'll have to get a lot more specific about what to do about it, and eventually code it.

**Comment by 0x62** [comment]
Perhaps there needs to be an element of risk:reward, so you could simulate higher risk behaviors for more food. A simple implementation could be food in the path of a bullet briefly has a higher value (either absolute or relative to the distance between the food and bullet).
